tests/options.custom-processing.test.js

```diff
@@ -13,6 +13,10 @@ function collect(value, previous) {
   return previous.concat([value]);
 }
 
+function collectVoid(value, previous) {
```

Comment:
I think this is not a valid custom processing function since it does not return a value, so I don't want to check that it fails in the "expected" way. Im articular, if we change the implementation in the future in a way that changes this behaviour, I won't consider it a breaking  change.

Reply:
so I remove this test ?

Reply:
Dropted.